sct.py

```diff
@@ -441,6 +443,79 @@ def output_conf(config_files, backend):
     sys.exit(0)
 
 
+def _run_yaml_test(backend, full_path, env):
+    output = []
+    error = False
+    output.append(f'---- linting: {full_path} -----')
+    while os.environ:
+        os.environ.popitem()
+    for key, value in env.items():
+        os.environ[key] = value
+    os.environ['SCT_CLUSTER_BACKEND'] = backend
+    os.environ['SCT_CONFIG_FILES'] = full_path
+    logging.getLogger().handlers = []
+    logging.getLogger().disabled = True
+    config = SCTConfiguration()
+    try:
+        config.verify_configuration()
+        config.check_required_files()
+    except Exception as exc:  # pylint: disable=broad-except
+        output.append(''.join(traceback.format_exception(type(exc), exc, exc.__traceback__)))
+        error = True
+    return error, output
+
+
+@cli.command(help="Run test yaml configurations")
+@click.option('-b', '--backend', type=click.Choice(SCTConfiguration.available_backends), default='aws')
+@click.option('-i', '--include', type=str, default='')
+@click.option('-e', '--exclude', type=str, default='')
+def lint_yamls(backend, exclude: str, include: str):
+    if not include:
+        raise ValueError('You did not provide include filters')
+
+    exclude_filters = []
+    for flt in exclude.split(','):
+        if not flt:
+            continue
+        try:
+            exclude_filters.append(re.compile(flt))
+        except Exception as exc:
+            raise ValueError(f'Exclude filter "{flt}" compiling failed with: {exc}')
+
+    include_filters = []
+    for flt in include.split(','):
+        if not flt:
+            continue
+        try:
+            include_filters.append(re.compile(flt))
+        except Exception as exc:
+            raise ValueError(f'Include filter "{flt}" compiling failed with: {exc}')
+
+    original_env = {**os.environ}
+    pp = ProcessPoolExecutor(max_workers=5)
+
+    features = []
+    for root, base, files in os.walk('./test-cases'):
+        for file in files:
+            full_path = os.path.join(root, file)
+            if not any((flt.search(file) or flt.search(full_path) for flt in include_filters)):
+                continue
+            if any((flt.search(file) or flt.search(full_path) for flt in exclude_filters)):
+                continue
+            features.append(pp.submit(_run_yaml_test, backend, full_path, original_env))
+
+    error_had_happened = False
```

Comment:
Why is it named in past perfect tense? Just "failed = False" would be more correct I guess.